georocket-server/src/main/java/io/georocket/index/elasticsearch/ElasticsearchClient.java

```diff
@@ -111,12 +111,27 @@ public void close() {
    */
   public Observable<JsonObject> beginScroll(JsonObject query, int pageSize,
       String timeout) {
+    return beginScroll(query, new JsonObject(), pageSize, timeout);
+  }
+
+  /**
+   * Perform a search and start scrolling over the result documents
```

Comment:
Please mention the possibility to use an aggregation to differentiate this method from the other `beginScroll()` method.